src/app/shared/validationMessages.model.ts

```diff
@@ -33,6 +33,12 @@ export const formErrors = {
   'customDockerRegistryPath': ''
 };
 
+export const examplePatterns = {
```

Comment:
exampleDescriptorPatterns or placeholderDescriptorPatterns might be more descriptive

Reply:
I mentioned descriptorPatterns because 
```
'cwl': 'e.g. /Dockstore.cwl',
'wdl': 'e.g. /Dockstore.wdl',
'nextflow': 'e.g. /nextflow.config'
```
seems to only contains descriptors right now.  If it will later encompass many more placeholders of different variety, then it would be fine to leave as is.  Same idea would apply to the validationPatterns naming below.